test/CMakeLists.txt

```diff
@@ -353,6 +354,7 @@ foreach(SDK ${SWIFT_SDKS})
              (test_subset STREQUAL "validation") OR
              (test_subset STREQUAL "only_long") OR
              (test_subset STREQUAL "only_stress") OR
+             (test_subset STREQUAL "only_early_swiftdriver") OR
```

Comment:
Shouldn't you also need to add only_early_swiftdriver to the if statement on 361?

Reply:
I do not think so. Here, depending on which test subset target is being built, we are adding a list of which test directories need to be passed to `lit`. For the `only_early_swiftdriver` subset, we do need the `test_bin_dir`, because that subset only runs tests inside `test/Driver/`. We do not need to add the `validation_test_bin_dir` for this subset, because it does not require any of the tests from that directory. 

The same build may also be performing validation testing, but that will be built as a different target, with a different subset, which will include test directories relevant to it.

Reply:
alright. I am fine with this.